site/e2e/tests/auditLogs.spec.ts

```diff
@@ -0,0 +1,69 @@
+import { expect, test } from "@playwright/test";
+import {
+  createTemplate,
+  createWorkspace,
+  requiresEnterpriseLicense,
+} from "../helpers";
+import { beforeCoderTest } from "../hooks";
+
+test.beforeEach(async ({ page }) => await beforeCoderTest(page));
```

Comment:
Could this be rewritten as
```ts
test.beforeEach(({ page }) => beforeCoderTest(page));
```
?

Right now, it looks like the code is awaiting the result of `beforeCoderTest`, and then immediately throwing it back into another promise via the async function